playbooks/common/openshift-cluster/upgrades/upgrade_control_plane.yml

```diff
@@ -238,29 +238,22 @@
   any_errors_fatal: true
 
   pre_tasks:
+  - name: Load lib_openshift modules
```

Comment:
I believe you should be able to use any modules defined in `roles` in post_tasks.